xapian-letor/api/letor.cc

```diff
@@ -73,13 +72,13 @@ Letor::prepare_training_file(const string & query_file, const string & qrel_file
 }
 
 void
-Letor::create_ranker(int ranker_type, int metric_type) {
-    switch(ranker_type) {
-        case 0: internal->ranker = * new SVMRanker;
-                break;
-        case 1: break;
-        default: ;//cout<<"Please specify proper ranker.";
-    }
+Letor::create_ranker(int ranker_type, int metric_type) { //TODO: update when adding rankers
```

Comment:
I agree. Marking this as a todo. Will get this done for both rankers and scorers, while integrating ListNET and NDCGScore.